deploy/crds/apps.openshift.io_servicebindingrequests_crd.yaml

```diff
@@ -36,7 +36,7 @@ spec:
           properties:
             applicationSelector:
               description: ApplicationSelector is used to identify the application
-                connecting to the backing service operator.
+                connecting to the backing service operator. Deprecated.
```

Comment:
I would move `Deprecated` message upfront and make more descriptive, like when, and how to migrate, etc. So at the end of the paragraph, many would overlook it.

Reply:
Added a long message, check?